src/api/telemetry/TelemetryAPI.js

```diff
@@ -54,6 +55,23 @@ import TelemetryValueFormatter from './TelemetryValueFormatter.js';
  * @memberof module:openmct.TelemetryAPI~
  */
 
+/**
+ * Describes and bounds requests for telemetry data.
+ *
+ * @typedef TelemetrySubscriptionOptions
+ * @property {String} [strategy] symbolic identifier directing providers on how
```

Comment:
should we make this an enum? e.g., [ANNOTATION_TYPES](https://github.com/nasa/openmct/blob/master/src/api/annotation/AnnotationAPI.js#L36)

Reply:
Great idea, will do.

Reply:
Done.